PubnativeLite/PubnativeLiteDemo/Adapters/AppMonet Adapters/DFP Header Bidding Adapter/AMCustomEventInterstitial.m

```diff
@@ -0,0 +1,27 @@
+//
+//  Copyright © 2020 PubNative. All rights reserved.
+//
+//  Permission is hereby granted, free of charge, to any person obtaining a copy
+//  of this software and associated documentation files (the "Software"), to deal
+//  in the Software without restriction, including without limitation the rights
+//  to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+//  copies of the Software, and to permit persons to whom the Software is
+//  furnished to do so, subject to the following conditions:
+//
+//  The above copyright notice and this permission notice shall be included in
+//  all copies or substantial portions of the Software.
+//
+//  THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+//  IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+//  FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+//  AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+//  LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+//  OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+//  THE SOFTWARE.
+//
+
+#import "AMCustomEventInterstitial.h"
+
+@implementation AMCustomEventInterstitial
```

Comment:
Here you need the same change that @can-soykarafakili  added for MoPub. Since the bids are injected in a different way in AppMonet SDK as they are in HyBid

Reply:
We have a separate for adapter for AdMob. But I think you're talking about something different @eros902002?

Reply:
Fair enough. With the method of **add bids** the ad will be in the cache already so we could use the adapter as it is I think.

Reply:
Both approaches are fine by me. 👍  if that makes our work easier, we can remove the separate AdMob adapter and add that into the DFP Adapter. Or we can keep as it is. 😄

Reply:
But in AdMob we only have a banner adapter, at least we have to add that approach for the Interstitial i think @eros902002.

Reply:
@eros902002 I haven't touched those files. They were in the branch already. I was focused only on DFP part :)

Reply:
@orkhanalizada2020 Let's wait the integration test results from @faresbh . If AdMob and DFP / GAM adapters are working fine with the current state, then we're okay 👍 Otherwise, we can do the changes like @eros902002 said, (Merging AdMob & DFP adapters together like we did in MoPub Bidder & Mediation). How that sounds guys?